cafebabel/tests/test_articles.py

```diff
@@ -31,7 +31,7 @@ def test_article_with_non_ascii_title(published_article):
 
 
 def test_published_article_should_display_content(client, published_article,
-                                                  user):
+                                                  user, twitter_nickname):
```

Comment:
`twitter_nickname` is not something that exists.
That breaks the tests

Reply:
same